bin/install

```diff
@@ -73,7 +73,7 @@ downloaded_file_path() {
 download_url() {
   local version=$1
   local variant=$(get_variant)
-  echo "https://github.com/oracle/graal/releases/download/vm-${version}/graalvm-ce-${version}-${variant}.tar.gz"
+  echo "https://github.com/oracle/graal/releases/download/vm-${version}/graalvm-ce-${variant}-${version}.tar.gz"
```

Comment:
This will work for 19.0.0 but break all the older RC builds. For macOS this is also not enough, since they also switched from e.g. `graalvm-ce-1.0.0-rc16-macos-amd64.tar.gz` to `graalvm-ce-darwin-amd64-19.0.0.tar.gz` (notice the `darwin` instead of `macos`).